frontend/src/components/common/Autocomplete.tsx

```diff
@@ -36,14 +41,25 @@ export function NamespacesAutocomplete() {
     );
   }
 
+  React.useEffect(() => {
+    const selectedNamespace = getFilterValueByNameFromURL('namespace', location, true);
+    if (!selectedNamespace) {
+      return;
+    }
+    dispatch(setNamespaceFilter(selectedNamespace));
+  },
+  // eslint-disable-next-line
+  []);
+
```

Comment:
I am not sure this is the best place for this logic, since this means that if we access the cluster's root URL with namespaces, example: http://localhost:3000/c/acela/?namespace=dex,gangway
and then we move to the Workloads, the namespaces filtering is not applied (because the link in the sidebar won't have it).
However, if we choose the namespaces (e.g. from workloads) and go to the cluster view (which, BTW, will show an empty list of events as it's using the namespaces for filtering) and then click the Workloads view again, it will apply the filters.